virt/virt-4-10-release-notes.adoc

```diff
@@ -72,22 +72,28 @@ The SVVP Certification applies to:
 [id="virt-4-10-storage-new"]
 === Storage
 
+//CNV-16673 New CSI driver for HPP
+* You can use the xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[Kubernetes Container Storage Interface (CSI) driver] with the hostpath provisioner (HPP) feature to configure local storage for your virtual machines. By creating an HPP custom resource that includes one or more storage pools, as well as creating a storage class for the CSI driver, you can run the HPP with less disruption to your existing {product-title} nodes and clusters.
```

Comment:
Question: Why do we need this separate release note if we are already documenting the new functionality in the `Deprecated features` section? There is a lot of content overlap.

Reply:
Per our slack chat, the deprecation is a side effect of the new feature, not the purpose of the new feature.